cddagl/constants.py

```diff
@@ -474,7 +582,9 @@
     '960140f7926267b56ef6933670b7a73d00087bd53149e9e63c48a8631cfbed53': '0.F-1',
     'c87f226d8b4e6543fbc8527d645cf4342b5e1036e94e16920381d7e5b5b9e34f': '0.F-1',
     '5da7ebd7ab07ebf755e445440210309eda0ae8f5924026d401b9eb5c52c5b6e7': '0.F-2',
-    '6870353e6d142735dfd21dec1eaf6b39af088daf5eef27b02e53ebb1c9eca684': '0.F-2'
+    '6870353e6d142735dfd21dec1eaf6b39af088daf5eef27b02e53ebb1c9eca684': '0.F-2',
+    '3e0b15543015389c34ad679a931186a1264dbccb010b813f63b6caef2d158dc8': '0.F-3',
```

Comment:
Correct me if I'm wrong but you should put the SHA256 for the 64 bit version of 0.F-3, AFTER the x86 one. It seems they are inverted

Reply:
From what I could see of the code (that I saw) that uses this, it shouldn't matter what order they're in - it's a search using the hash as the key and is used to identify the installed version without regard for whether it's 32- or 64-bit.

Reply:
Yes, I guess you're right, order does not matter, I guess I should have checked twice; in fact it does work for me too